communication/slack-config/usergroups.yaml

```diff
@@ -51,13 +51,14 @@ usergroups:
       - release-management
       - sig-release
     members:
+      - Balaji
```

Comment:
This still needs to be `SaranBalaji90`.  `communication/slack-config/users.yaml` defines the mapping between GH handle and slack ID (that file needs to be fixed too).

Reply:
Correct. The most important thing though is they need to match

Reply:
Ah, got it. Fixed.